app/src/Homepage/Homepage-components.tsx

```diff
@@ -42,6 +42,9 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
     );
 }
 
+export const GetStringAfterSpace = ( label : string ) => {
```

Comment:
A little too vague of a name. You should name it how you are using it so like "GetClassNumber" would be better

Reply:
changed